app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -36,208 +36,122 @@ keywords:
   
 ---
 
-If you’re applying for teacher training as a non-UK citizen, you will need a visa or immigration status allowing you to study or work in England. 
+If you’re applying for teacher training as a non-UK citizen, you will need a [visa or immigration status allowing you to study or work in England](#check-your-status). 
 
-The information on this page is for:
+## How to apply for your visa
 
-* non-UK citizens living in the UK
-* non-UK citizens living outside the UK
-* refugees or asylum seekers
+Most candidates who don't already have a visa or immigration status allowing them to study or work in England will need to apply for a Student or Skilled worker visa.
 
-You will not need a visa to train to teach in the UK if you:
+### Student visa 
 
-* are an Irish citizen
-* have settled or pre-settled status under the [EU Settlement Scheme (EUSS)](https://www.gov.uk/settled-status-eu-citizens-families)
-* have [indefinite leave to remain](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk) 
+Most teacher training courses in England are fee-paying. You will need a Student visa if you train on a fee-paying course. Your Student visa must be sponsored by your teacher training provider. 
 
-If you are currently studying in the UK on a student visa, you may be able to apply for a [graduate visa](#graduate-visa).
+### Skilled worker visa 
```

Comment:
This heading isn't aligned properly on the demo app - it looks like there's an extra space before the H3?

Also, small point so feel free to ignore - but should this be 'Student Worker visa' to align with GOV.UK?

Reply:
Skilled Worker? Yes good spot. Not sure why it is not aligning as no space, @martyn-w do you know why this might be happening?